docs/source/_templates/index.html

```diff
@@ -1,1730 +1,215 @@
+                        <blockquote>Jill preps RNA libraries and monitors rRNA content</blockquote>
+											</section>
+										</div>
+									</div>
+								</div>
+								<ul class="actions actions-centered">
+									<li><a href="#" class="button style1 big">Live Demo</a></li>
+									<li><a href="#" class="button style2 big">Get The Code</a></li>
+								</ul>
+							</section>
+
+					</div>
+				</div>
+
+			<!-- Highlights -->
+				<div class="wrapper style3">
+					<div class="title">Getting Started</div>
+					<div id="highlights" class="container">
+            <header class="style1">
+              <h2>Are you familiar with MultiQC?</h2>
+              <p>
+                MegaQC relies on MultiQC; it's little brother.<br class="mobile-hide" />
+                If you're not sure what that is, then check out
+                <a href="http://multiqc.info/">http://multiqc.info</a> first&hellip;
+              </p>
+            </header>
+						<div class="row 150%">
+							<div class="4u 12u(mobile)">
+								<section class="highlight">
+									<a href="#" class="image featured"><img src="https://picsum.photos/326/166" alt="" /></a>
+									<h3><a href="#">Overview of features</a></h3>
+									<p>
+                    A quick tour of what MegaQC can do: see the highlights and get the postcard!
+                    <br><em>[34s]</em>
+                  </p>
+								</section>
+							</div>
+							<div class="4u 12u(mobile)">
+								<section class="highlight">
+									<a href="#" class="image featured"><img src="https://picsum.photos/326/167" alt="" /></a>
+									<h3><a href="#">Basic usage</a></h3>
+									<p>
+                    Screencast of logging in for the first time, uploading data and creating some plots
+                    <br><em>[4m 24s]</em>
+                  </p>
+								</section>
+							</div>
+							<div class="4u 12u(mobile)">
+								<section class="highlight">
+									<a href="#" class="image featured"><img src="https://picsum.photos/326/165" alt="" /></a>
+									<h3><a href="#">Installation instructions</a></h3>
+									<p>
+                    An overview of how to install MegaQC - from personal laptop to production server.
+                    <br><em>[6m 49s]</em>
+                  </p>
+								</section>
+							</div>
+						</div>
+					</div>
+				</div>
+
+			<!-- Footer -->
+				<div id="footer-wrapper" class="wrapper">
+					<div class="title">Try before you buy</div>
+					<div id="footer" class="container">
+						<header class="style1">
+							<h2>Still not sure?</h2>
+							<p>
+								Investing time into a new tool is a bit of a commitment, so to be sure <br class="mobile-hide" />
+                that MegaQC does what you want you can check out some further resources:
+							</p>
+              <ul class="actions">
+                <li><a href="#" class="button style2">Read the docs</a></li>
```

Comment:
This button should lead to 
```suggestion
                <li><a href="docs/index.html" class="button style2">Read the docs</a></li>
```